changelog/fix_better_handle_prism_loaderror.md

```diff
@@ -0,0 +1 @@
+* [#349](https://github.com/rubocop/rubocop-ast/pull/349): Show better error messages when `prism` is too old or linked to the wrong Ruby version. ([@earlopain][])
```

Comment:
Since the behavior of the feature for users doesn't change, the changelog entry is likely unnecessary. Can you remove it?